cubix-tree-sitter/src/AST/GenerateSyntax.hs

```diff
@@ -0,0 +1,180 @@
+{-# LANGUAGE TemplateHaskell #-}
+module AST.GenerateSyntax where
```

Comment:
This looks like a file where only a fraction of declarations should be imported.

Reply:
@jkopanski This file is un-sectioned and has no enmuerated exports. I can't tell what the starting point for this file is. Please address.